src/applications/gi-sandbox/containers/SearchPage.jsx

```diff
@@ -9,18 +9,36 @@ import { useHistory } from 'react-router-dom';
 import CompareDrawer from './CompareDrawer';
 import NameSearchResults from '../containers/search/NameSearchResults';
 import LocationSearchResults from '../containers/search/LocationSearchResults';
+import NameSearchForm from './search/NameSearchForm';
+import LocationSearchForm from './search/LocationSearchForm';
 
 export function SearchPage({
   dispatchChangeSearchTab,
   dispatchSetPageTitle,
   search,
 }) {
-  useEffect(() => {
-    dispatchSetPageTitle(`${PAGE_TITLE}: VA.gov`);
-  }, []);
   const queryParams = useQueryParams();
   const history = useHistory();
   const { tab, error } = search;
+  const [smallScreen, setSmallScreen] = useState(
+    matchMedia('(max-width: 600px)').matches,
+  );
+
+  useEffect(
+    () => {
+      dispatchSetPageTitle(`${PAGE_TITLE}: VA.gov`);
+    },
+    [dispatchSetPageTitle],
+  );
+
+  useEffect(() => {
+    const checkSize = () => {
+      setSmallScreen(matchMedia('(max-width: 480px)').matches);
```

Comment:
Should these two max-widths be the same?

Reply:
Yes, thanks!